src/python/dxpy/bindings/dxjob.py

```diff
@@ -179,11 +184,12 @@ def new(self, fn_input, fn_name, name=None, tags=None, properties=None, details=
                 req_input["tags"] = tags
             if properties is not None:
                 req_input["properties"] = properties
-            if instance_type is not None or cluster_spec is not None or fpga_driver is not None:
+            if instance_type is not None or cluster_spec is not None or fpga_driver is not None or nvidia_driver is not None:
```

Comment:
I would personally use the same way with `any` as above

Reply:
agree, fixed.